doc/developer_guide/coding-style.rst

```diff
@@ -0,0 +1,120 @@
+.. _coding-style:
+
+Coding and Documentation Style
+******************************
+
+Documentation Style
+###################
+When writing documentation, each reference to a PSyclone class or function
+should be set in italic, i.e. using a single backtick. except in headings
```

Comment:
Done - I actually put the "i.e. .. backtick" in (), otherwise I found the sentence ambiguous (I think it could mean set it in italic without backticks in headings)

Reply:
Thanks. I've tweaked it slightly.